blockchain/My-Wallet-V3-iOS
# feat(KYC): IOS-1161 move KYC responsibility to Exchange coordinator

## Objective

Move the responsibility of whether or not the KYC flow or the exchange flow should be started to the ExchangeCoordinator.

## Description

Migrated the KYC flow logic to the Exchange coordinator and made the KYC coordinator a singleton.

## How to Test

While in the `dev` environment, tap on the `Exchange` item in the sidebar. Observe that the KYC flow is initialized for a user with an account status of unverified. Then observe that the exchange flow is started for a user with an account status of verified.

## Merge Checklist
